pkg/minikube/constants/constants.go

```diff
@@ -29,8 +29,9 @@ const MachineName = "minikubeVM"
 // APIServerPort is the port that the API server should listen on.
 const APIServerPort = 8443
 
-// Fix for windows
-var Minipath = filepath.Join(homedir.HomeDir(), ".minikube")
+// Fix for windows: the home dir contains forward slashes, which results
```

Comment:
Would you mind taking a look at #411 to see if it still fixes the problem you were seeing? Just moved where we are converting all slashes to forward slashes to ensure file paths are still file paths where they're needed & only converted to forward slashes when we need them to be for URIs.